lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1065,15 +1089,28 @@ int mme_app_handle_create_sess_resp(
     if (
       create_sess_resp_pP->bearer_contexts_created.bearer_contexts[i]
         .cause.cause_value != REQUEST_ACCEPTED) {
-      DevMessage(
+      OAILOG_ERROR(
+        LOG_MME_APP,
         "Cases where bearer cause != REQUEST_ACCEPTED are not handled\n");
     }
-    DevAssert(
+    if (
       create_sess_resp_pP->bearer_contexts_created.bearer_contexts[i]
-        .s1u_sgw_fteid.interface_type == S1_U_SGW_GTP_U);
+        .s1u_sgw_fteid.interface_type != S1_U_SGW_GTP_U) {
+      OAILOG_ERROR(
+        LOG_MME_APP,
+        "Invalid S1U SGW GTP F-TEID interface type: %d (Expected: %d)\n",
+        create_sess_resp_pP->bearer_contexts_created.bearer_contexts[i]
+          .s1u_sgw_fteid.interface_type,
+        S1_U_SGW_GTP_U);
+    }
 
     current_bearer_p = mme_app_get_bearer_context(ue_context_p, bearer_id);
-    AssertFatal(current_bearer_p, "Could not get bearer context");
+    if (current_bearer_p == NULL) {
+      OAILOG_ERROR(
+        LOG_MME_APP,
+        "Failed to get bearer context for bearer Id (%d)\n",
+        bearer_id);
+    }
```

Comment:
Continue the loop if no valid bearer_context is found

Reply:
The fix has been added. The loop will now continue to check for the next bearer in the loop, instead of exiting from the function. If not a single valid bearer is found after this loop, it will exit with a failure response.